vm-setup/roles/libvirt/defaults/main.yml

```diff
@@ -23,13 +23,27 @@ extradisks_list:
 extradisks_size: 8G
 
 # The packages required to set up our desired libvirt environment.
+# (Tested on Centos 7)
 libvirt_packages:
   - qemu-kvm
   - libvirt
   - libvirt-python
   - libguestfs-tools
   - python-lxml
   - polkit-pkla-compat
+  - python-netaddr
+  - python2-virtualbmc
+
+# We expect virtualbmc to already be installed on rhel8 as a pre-req to running this,
+# as there's no rhel package available yet.
+libvirt_packages_rhel8:
+  - qemu-kvm
+  - libvirt
+  - python3-libvirt
+  - libguestfs-tools
+  - python3-lxml
+  - polkit-pkla-compat
+  - python3-netaddr
```

Comment:
Where do we install virtualbmc in the el8 case?

Reply:
not done here, it's assuming it got installed from source before any of this runs.  In dev-scripts, it installs it in the first prereqs script.  We could do the same in the 01 script for metal3-dev env too, but I forgot to do it.

Reply:
Ack Ok not blocking on it, perhaps we should add it here though so this environment will also work on el8 - we can make that a follow-up though if you prefer.

Reply:
probably we should move all the install stuff from the scripts into ansible, but that's definitely a follow-up, I'll try to take a look at it.